tests/ui/specs/ContextualPopupDecorator/ContextualPopupDecorator-specs.js

```diff
@@ -0,0 +1,42 @@
+const Page = require('./ContextualPopupDecoratorPage');
+
+describe('ContextualPopupDecorator', function () {
+	beforeEach(function () {
+		Page.open();
+	});
+
+	const {
+		button1,
+		button2
+	} = Page.components;
+
+	describe('not using open', function () {
+		it('should focus the first button on start', function () {
+			expect(button1.self.isFocused()).to.be.true();
+		});
+
+		describe('using 5-way', function () {
```

Comment:
How about adding 'using pointer' case?
And also closing the contextual popup by clicking?

Reply:
Thanks for suggestion! I added some more tests.